sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/SupportsPushDownTopN.java

```diff
@@ -35,4 +35,12 @@
      * Pushes down top N to the data source.
      */
     boolean pushTopN(SortOrder[] orders, int limit);
+
+    /**
+     * Whether the top N is partially pushed or not. If it returns true, then Spark will do top N again.
+     * This method will only be called when `pushTopN` returns true.
+     *
+     * @return true if sort be pushed down to datasource partially, false otherwise.
```

Comment:
```suggestion
     * @return true if top N is pushed down to this data source partially, false otherwise.
```

Reply:
Actually, we can simply remove this `@return ...`. The method doc already describe the return value.

Reply:
OK